command_list.py

```diff
@@ -12,34 +12,42 @@
   commands_keys, \
   commands_mouse, \
   commands_pause, \
-  commands_external
+  commands_external, \
+  commands_subroutines, \
+  commands_dialog, \
+  commands_browser_automation, \
+  commands_file, \
+  commands_documentation
+
+# @@@ a test command set for testing things!  Will be removed for production
+try:
+    import commands_test
+except:
+    traceback.print_exc()
+    pass
 
 # This library could be considered optional, but is not platform specific
 try:
     import commands_rpncalc
 except ImportError:
-    print("[LPHK] WARNING: RPN_EVAL command is not available")  
+    print("[LPHK] WARNING: RPN_EVAL command is not available")

```

Comment:
This should be info instead of a warning

Reply:
Sure, but since it doesn't have any special dependencies, it's probably now better treating it like a core command set

Reply:
But that change will be committed shortly